src/components/ftb-stage-cup-net/ftb-cup-net-quadratic.component.tsx

```diff
@@ -225,54 +226,82 @@ export class FtbStageCupNetQuadratic {
     this.columns = [...columnsLeft, ...columnsRight];
   }
 
-  highlight(team: Team | null) {
-    writeTask(() => {
-      const gameHasTeam = (g: Game | null) => {
-        if (!g) return;
-        return g.home.team._id == team?._id || g.away.team._id == team?._id;
-      };
-
-      /** clear all highlighting first */
-      this.netLines.forEach(line => {
-        line.el.classList.remove('highlighted');
-        line.from.leftDotEl.classList.remove('highlighted');
-        line.from.rightDotEl.classList.remove('highlighted');
-
-        Array.from(line.from.leftDotEl.parentElement.querySelectorAll('.row')).forEach(r =>
-          r.classList.remove('highlighted'),
-        );
-        Array.from(line.to.leftDotEl.parentElement.querySelectorAll('.row')).forEach(r =>
-          r.classList.remove('highlighted'),
-        );
-        line.to.leftDotEl.classList.remove('highlighted');
-        line.to.rightDotEl.classList.remove('highlighted');
-      });
+  highlight(teams: Team[] | null) {
+    teams?.map(team => {
+      writeTask(() => {
```

Comment:
wrap all teams highlighting into one `writeTask`. Otherwise we will request new animation frame for each operation